cmd/dusk/server.go

```diff
@@ -176,7 +166,7 @@ func (s *Server) OnConnection(conn net.Conn, addr string) {
 // Close the chain and the connections created through the RPC bus
 func (s *Server) Close() {
 	// TODO: disconnect peers
-	s.chain.Close()
+	_ = s.chain.Close()
```

Comment:
For this and other lint errors, we can have a separate issue that fixes all lint issues but also enabled CI lint checks.

Reply:
sure, there is already #366 opened to address this. Leaving it unfixed in this PR seemed a bit weird. So I suggest we leave it fixed instead of undoing it. Would you agree ?

Reply:
Sure.